pkg/northbound/gnmi/set.go

```diff
@@ -280,6 +267,27 @@ func listenForDeviceResponse(changes mapNetworkChanges, target string, name stor
 	}
 }
 
+func doRollback(changes mapNetworkChanges, mgr *manager.Manager, target string,
+	name store.ConfigName, errResp error) error {
+	rolledbackIDs := make([]string, 0)
+	for configName := range changes {
+		changeID, err := mgr.RollbackTargetConfig(string(configName))
+		log.Infof("Rolling back %s on %s", store.B64(changeID), configName)
```

Comment:
log should happen after error check or before calling RollbackTargetConfig()

Reply:
done